### PR TITLE
logentries: Update to recent release and also add plist

### PR DESCRIPTION
**Pull request by jcftang**
This pull request updates the logentries agent to the latest stable version. It also adds a small sanity check and a plist file for running in a daemonised mode.

### CODE REVIEW

File: Library/Formula/logentries.rb [hunk 1]
```diff
@@ -20,4 +21,29 @@ def install
...
+        <true/>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+    EOS
+  end
+
+  test do
+    system "if [ \"`#{bin}/le --help || echo $?`\" -ne \"4\" ]; then exit 1; fi"
```
Comment (bfontaine) [inline Library/Formula/logentries.rb]: `shell_output("#{bin}/le --help", 4)` is what you’re looking for :wink: 

Can’t we write a test that does something more substantial than e.g. `--help`/`--version`?

Comment (jcftang) [inline Library/Formula/logentries.rb]: Unfortunately the agent needs user configuration before it does something meaningful, the --help/--version check is really the only thing that I can run to check that it works after an install.

I'll update that check and push it up.

Comment (mikemcquaid) [inline Library/Formula/logentries.rb]: You can write user configuration in the test.

Comment (jcftang) [inline Library/Formula/logentries.rb]: @mikemcquaid but that requires a log/account key to be valid for a config to actually run to validate it works. Of course I could do a failing test if that works?

Comment (StephenHynes7) [inline Library/Formula/logentries.rb]: @mikemcquaid the problem is that any sort of configuration will expose some private token to the world which isn't good from a security point of view (even if it is a throw away account). Any service which requires registration before it becomes active would have this problem. The other alternative is we can check to see if a more specific command works but we will still be doing a test expecting to see an error message.

File: Library/Formula/logentries.rb [hunk 2]
```diff
@@ -20,4 +21,29 @@ def install
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
+
+  def plist; <<-EOS.undent
```
Comment (mikemcquaid) [inline Library/Formula/logentries.rb]: Please also add a `plist_options` (like e.g. `mysql.rb`

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5b465b7c494784861d51bd654fd8828488e4f076. Thank you for your contribution to Homebrew @jcftang; we appreciate it! :smiley_cat:

**Pull request closed without merging**